cernopendata/modules/fixtures/data/articles/getting_started/md/cms.md

```diff
@@ -42,7 +42,7 @@ Excellent! You have successfully opened a CMS AOD file in ROOT. If this was the
 
 On the left window of ROOT (see the screenshot below), double-click on the file name (`root://eospublic.cern.ch//eos/opendata/...`). You should see a list of entries under `Events`, each corresponding to a collection of reconstructed data. We are interested in the collections containing information about reconstructed physics objects.
 
-<img src="http://opendata.cern.ch/img/docs/Screenshot_001_Tbrowser_t.png" width="70%">
+![](../../../../../theme/static/articles/cms/cms_tbrowser.png)
```

Comment:
Regarding file names, we can muse IRL about the best strategy. If we go for renaming, then perhaps we should make it more complete.  Here are some raw notes for illustration of my thoughts.

The current status in your PR is:

```
.../modules/fixtures/data/articles/getting_started/alice.json       |  22 +++++++
 .../modules/fixtures/data/articles/getting_started/lhcb.json        |  22 +++++++
 .../modules/fixtures/data/articles/getting_started/md/alice.md      |  32 ++++++++++
 .../modules/fixtures/data/articles/getting_started/md/cms.md        |   4 +-
 .../modules/fixtures/data/articles/getting_started/md/lhcb.md       |  77 +++++++++++++++++++++++
 cernopendata/modules/fixtures/data/articles/vms/alice.json          |  22 +++++++
 cernopendata/modules/fixtures/data/articles/vms/lhcb.json           |  22 +++++++
 cernopendata/modules/fixtures/data/articles/vms/md/alice.md         |  99 ++++++++++++++++++++++++++++++
 cernopendata/modules/fixtures/data/articles/vms/md/cms_2010.md      |  12 ++--
 cernopendata/modules/fixtures/data/articles/vms/md/lhcb.md          |  28 +++++++++
 cernopendata/modules/theme/static/articles/alice/1.png              | Bin 0 -> 9364 bytes
 cernopendata/modules/theme/static/articles/alice/10.png             | Bin 0 -> 12979 bytes
 cernopendata/modules/theme/static/articles/alice/11.png             | Bin 0 -> 6486 bytes
 cernopendata/modules/theme/static/articles/alice/2.png              | Bin 0 -> 41524 bytes
 cernopendata/modules/theme/static/articles/alice/3.png              | Bin 0 -> 53240 bytes
 cernopendata/modules/theme/static/articles/alice/4.png              | Bin 0 -> 72591 bytes
 cernopendata/modules/theme/static/articles/alice/5.png              | Bin 0 -> 45136 bytes
 cernopendata/modules/theme/static/articles/alice/6.png              | Bin 0 -> 5196 bytes
 cernopendata/modules/theme/static/articles/alice/7.png              | Bin 0 -> 11772 bytes
 cernopendata/modules/theme/static/articles/alice/8.png              | Bin 0 -> 65650 bytes
 cernopendata/modules/theme/static/articles/alice/9.png              | Bin 0 -> 23198 bytes
 cernopendata/modules/theme/static/articles/alice/get_started_1.png  | Bin 0 -> 55265 bytes
 cernopendata/modules/theme/static/articles/alice/get_started_2.png  | Bin 0 -> 63695 bytes
 cernopendata/modules/theme/static/articles/cms/cms_tbrowser.png     | Bin 0 -> 46848 bytes
 .../modules/theme/static/articles/lhcb/get_started_lhcb_1.png       | Bin 0 -> 52616 bytes
 .../modules/theme/static/articles/lhcb/get_started_lhcb_2.png       | Bin 0 -> 170215 bytes
 .../modules/theme/static/articles/lhcb/get_started_lhcb_3.png       | Bin 0 -> 37056 bytes
 cernopendata/modules/theme/static/articles/lhcb/vm_lhcb_1.png       | Bin 0 -> 20351 bytes
 cernopendata/modules/theme/static/articles/lhcb/vm_lhcb_2.png       | Bin 0 -> 25313 bytes
```

It introduces new directory (`/articles/`) and new names that is OT1H similar to the story, but OTOH not fully associated in a complete manner.  So it is not easy to find related stuff by grepping (get started, getting started, vms, etc.)

We have several options here:

Option 1: we keep the old PROD names (`img/docs/Screenshot_001_Tbrowser_t.png` or `/static/img/alice/1.png`) and we don't do any reogranisation. Just use old names and locations. While this may be practical for upgrades and old URLs, I'm afraid it can lead to a mess later.

Option 2: we reogranise as you nicely started, but we base ourselves on the SLUG of where the article will be published. (E.g. `http://0.0.0.0/articles/getting-started-with-alice-open-data`.) This will help us to find associated files very easily, and will lead to hopefully coherent file organisation structure.

For example:

```
cernopendata/modules/fixtures/data/articles/getting-started-with-alice-open-data/getting-started-with-alice-open-data.json
cernopendata/modules/fixtures/data/articles/getting-started-with-alice-open-data/getting-started-with-alice-open-data.md
cernopendata/modules/theme/static/articles/getting-started-with-alice-open-data/1.png
cernopendata/modules/theme/static/articles/getting-started-with-alice-open-data/2.png
[...]

cernopendata/modules/fixtures/data/articles/cms-2011-virtual-machines-how-to-install/cms-2011-virtual-machines-how-to-install.json
cernopendata/modules/fixtures/data/articles/cms-2011-virtual-machines-how-to-install/cms-2011-virtual-machines-how-to-install.md
cernopendata/modules/theme/static/articles/cms-2011-virtual-machines-how-to-install/some-picture.png
cernopendata/modules/theme/static/articles/cms-2011-virtual-machines-how-to-install/other-picture.png
```

This might necessitate small amendments in the code, but it may be better overall in the longer term when we shall have hundreds of articles.

I would also profit from the occasion think about *shorter* slugs; perhaps they could be taken directly from the filename `articles/*.json`.  E.g. for `articles/my-first-article.json` the slug would be `my-first-article`. We could even store it in the record, since it's a kind of PID, just like "recid". CC @pamfilos 

BTW also beware that we may change "/articles/my-first-article" into "/post/my-first-article". CC @daslerr 

Let's muse IRL today?

Reply:
ok great 👍🏻  I amended the new file organisation structure.

Reply:
Yes, per our conversations today, we've decided to change all of the articles that aren't news to ``/docs``. (see #1312) 

I also agree that reorganizing (option 2 above) with shorter slugs taken from the file name sounds like a good way to go. Perhaps things like ``/getting-started-alice`` or ``/cms-vm-2011`` if you're looking for specific shortening suggestions. :)